integreat_cms/static/src/js/media-management/component/edit-sidebar.tsx

```diff
@@ -221,6 +229,19 @@ const EditSidebar = ({
                         <label className="secondary my-0">{mediaTranslations.label_file_modified}</label>
                         <p>{file.lastModified}</p>
                     </div>
+                    <div className="flex flex-wrap justify-between gap-2 hover:bg-gray-50 p-4 border-b">
+                        <label className="secondary my-0">{mediaTranslations.label_file_is_hidden}</label>
+                        <input
+                            id="file-hide-input"
+                            name="is_hidden"
+                            type="checkbox"
+                            value={String(changedFile.isHidden)}
+                            checked={changedFile.isHidden}
+                            onClick={() => {
+                                onIsHiddenCheckboxClick();
+                            }}
+                        />
+                    </div>
```

Comment:
```suggestion
                    {file.isGlobal && (
                        <div className="flex flex-wrap justify-between gap-2 hover:bg-gray-50 p-4 border-b">
                            <label className="secondary my-0">{mediaTranslations.label_file_is_hidden}</label>
                            <input
                                id="file-hide-input"
                                name="is_hidden"
                                type="checkbox"
                                value={String(changedFile.isHidden)}
                                checked={changedFile.isHidden}
                                onClick={() => {
                                    onIsHiddenCheckboxClick();
                                }}
                            />
                        </div>
                    )}
```

For region files too, just like region directories.

Reply:
Oh, AFAIK is the checkbox only supposed to be visible for global dirs and files. So it was a mistake on my side to add it to regional directories. I will remove it there